test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -76,13 +73,12 @@ proc main() {
 //
 // Scan the alphabets' probabilities to compute cut-offs
 //
-proc sumAndScale(alphabet: [?D]) {
+proc sumProbs(alphabet: [?D]) {
```

Comment:
It looks like the ': [?D]' could be dropped now that we're not referring to 'D' within this routine?